dashboard/app/helpers/levels_helper.rb

```diff
@@ -570,6 +570,9 @@ def blockly_options
     level_options['puzzle_number'] = script_level ? script_level.position : 1
     level_options['lesson_total'] = script_level ? script_level.lesson_total : 1
     level_options['final_level'] = script_level.final_level? if script_level
+    level_options['lastLevelInLesson'] = script_level.end_of_lesson? if script_level
+    level_options['lastLevelInScript'] = script_level.end_of_script? if script_level
+    level_options[:showEndOfLessonMsgs] = true
```

Comment:
- [ ] figure out how to work around needing to check the user experiment for this setting on script, maybe bypass and ship w/o flag?